govuk_chat_evaluation/jailbreak_guardrails/generate.py

```diff
@@ -0,0 +1,40 @@
+import asyncio
+from pathlib import Path
+from ..dataset_generation import generate_dataset, run_rake_task
+from ..file_system import jsonl_to_models, write_generated_to_output
+from .evaluate import EvaluationResult
+from pydantic import BaseModel
+from typing import List
+
+
+class GenerateInput(BaseModel):
+    question: str
+    expected_outcome: bool
+
+
+def generate_and_write_dataset(input_path: str, provider: str, output_dir: Path):
+    models = jsonl_to_models(Path(input_path), GenerateInput)
+    generated = generate_models_to_evaluation_models(provider, models)
+    return write_generated_to_output(output_dir, generated)
+
+
+def generate_models_to_evaluation_models(
```

Comment:
[comment] I find the choice of name for this function a bit confusing. See comments below about "models".

Reply:
Addressed in https://github.com/kevindew/govuk-chat-evaluation-prototype/pull/2#discussion_r2003799238